sql/sql_acl.cc

```diff
@@ -558,22 +558,22 @@ TABLE_FIELD_TYPE mysql_tables_priv_table_fields[MYSQL_TABLES_PRIV_FIELD_COUNT] =
 const TABLE_FIELD_DEF
   mysql_db_table_def= {MYSQL_DB_FIELD_COUNT, mysql_db_table_fields};
 
-const TABLE_FIELD_DEF
+__attribute__((unused)) const TABLE_FIELD_DEF
```

Comment:
In what configurations this gets skipped? NO_EMBEDDED_ACCESS_CHECKS? Can't you just use the same ifdef here instead of marking as unused?

If fixed in the current way, then MY_ATTRIBUTE everywhere

Reply:
MY_ATTRIBUTE was introduced in 5.6, 5.5 used simply attribute (there is an ifdef for it in include/my_attribute.h)

Yes, it's that definition, I could add that ifdef here too.

Reply:
Please use that definition (but my comment of taking higher version fixes, if available, takes precedence)